interactive/templatetags/selected_filter.py

```diff
@@ -0,0 +1,34 @@
+from django import template
+
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True)
+def is_filter_selected(context, *, key, value, **kwargs):
```

Comment:
This also isn't being used right now, although maybe that will change once some filters are added?

Reply:
Yep :)